src/Components/Starred/Starred.jsx

```diff
@@ -1,17 +1,24 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import Post from '../Post/Post';
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
+import { getAllStarred } from '../../Reducers/userReducer';
 
 const Starred = () => {
+  const dispatch = useDispatch();
   const { starred } = useSelector((state) => state.user);
+  const { posts } = useSelector((state) => state.post);
+
+  useEffect(() => {
+    dispatch(getAllStarred());
+  }, [dispatch, posts]);
```

Comment:
Suggestion :- Just adding posts in dependency array will work right ? What is the purpose of adding dispatch too

Reply:
Just to get rid of console warnings.

Reply:
Also can you please approve the changes?

Reply:
Okay got it